tools/test-npm.sh

```diff
@@ -29,10 +29,10 @@ export npm_config_prefix="npm-prefix"
 export npm_config_tmp="npm-tmp"
 
 # install npm devDependencies and run npm's tests
-
-../$NODE cli.js install --ignore-scripts
-../$NODE cli.js run-script test-legacy
-../$NODE cli.js run-script test
+NODEPATH="$(../$NODE -e 'console.log(require("path").resolve(".."))')"
```

Comment:
Yeah, it should, as it executes all other instances w/ the `NODE` env var set by npm based on `process.execPath`